types/monome-grid/monome-grid-tests.ts

```diff
@@ -0,0 +1,15 @@
+import monomeGrid from "monome-grid";
```

Comment:
Does the underlying library have a property called `monomeGrid`?  Is this going to fail at runtime?